src/components/timeout-dialog/__tests__/timeout-dialog-can-be-shown-multiple-times.test.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * @jest-environment ./lib/puppeteer/environment.js
+ */
+
+/* eslint-env jest */
+/* eslint-disable no-underscore-dangle */
+
+import path from 'path';
+import browserify from 'browserify';
+
+function packageScriptForBrowser(script) {
+  return new Promise((resolve, reject) => {
+    browserify(script).bundle((err, src) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(src.toString('utf8'));
+      }
+    });
+  });
+}
+
+async function installFakeTimersOnLoad(page, initialSecondsOffset) {
+  await page.evaluateOnNewDocument(
+    await packageScriptForBrowser(path.join(__dirname, '__helpers__', 'install-fake-timers.js')),
+  );
+  if (initialSecondsOffset) {
+    await page.evaluateOnNewDocument(
+      (ms) => window.clock.tick(ms),
+      initialSecondsOffset * 1000,
+    );
+  }
+  return async function advanceClock(seconds) {
+    await page.evaluate(
+      (ms) => window.clock.tick(ms),
+      seconds * 1000,
+    );
+  };
+}
+
+async function serveFakePage(page, url, body) {
+  await page.setRequestInterception(true);
+  page.on('request', (req) => {
+    if (req.url() === url) {
+      req.respond({
+        contentType: 'text/html',
+        body,
+      });
+    } else {
+      req.continue();
+    }
+  });
+}
```

Comment:
not used anywhere else at the moment but could move these into __helpers__

Reply:
thinking about it some more, might be better to rename `__helpers__` to `__test-helpers__` and move this test file alongside the other test files. Could refactor our tests into `__tests__` folder in future, but at the moment we have convention of having them inline